requirements.txt

```diff
@@ -1,11 +1,7 @@
 praw==4.4.0
 redis<3.0.0
-hiredis
```

Comment:
https://github.com/redis/hiredis-py It's primarily for pipeline'd redis commands, which we don't actually use (although we might need to as we move more towards the daemonized side of things). I'm open to removing it until we actually need it.